recipe/meta.yaml

```diff
@@ -26,18 +26,18 @@ build:
     - mv lib/{{ cuda_compiler_version }}/libcutensor.so* $PREFIX/lib/     # [unix and cuda_compiler_version in ("10.1", "10.2")]
     - mv lib/11/libcutensor.so* $PREFIX/lib/                              # [unix and cuda_compiler_version in ("11.0", )]
 
-    - 7z x %SRC_DIR%\\libcutensor_1.2.2.exe                                                                     # [win64]
-    - copy %SRC_DIR%\\libcutensor\\libcutensor\\include\\cutensor.h %LIBRARY_INC%\\                             # [win64]
-    - mkdir %LIBRARY_INC%\\cutensor                                                                             # [win64]
-    - copy %SRC_DIR%\\libcutensor\\libcutensor\\include\\cutensor\\types.h %LIBRARY_INC%\\cutensor              # [win64]
-    - copy %SRC_DIR%\\libcutensor\\libcutensor\\lib\\{{ cuda_compiler_version }}\\cutensor.dll %LIBRARY_BIN%\\  # [win64 and cuda_compiler_version in ("10.1", "10.2")]
-    - copy %SRC_DIR%\\libcutensor\\libcutensor\\lib\\{{ cuda_compiler_version }}\\cutensor.lib %LIBRARY_LIB%\\  # [win64 and cuda_compiler_version in ("10.1", "10.2")]
-    - copy %SRC_DIR%\\libcutensor\\libcutensor\\lib\\11\\cutensor.dll %LIBRARY_BIN%\\                           # [win64 and cuda_compiler_version in ("11.0", )]
-    - copy %SRC_DIR%\\libcutensor\\libcutensor\\lib\\11\\cutensor.lib %LIBRARY_LIB%\\                           # [win64 and cuda_compiler_version in ("11.0", )]
+    - copy include\\cutensor.h %LIBRARY_INC%\\                             # [win64]
+    - mkdir %LIBRARY_INC%\\cutensor                                        # [win64]
+    - copy include\\cutensor\\types.h %LIBRARY_INC%\\cutensor              # [win64]
+    - copy lib\\{{ cuda_compiler_version }}\\cutensor.dll %LIBRARY_BIN%\\  # [win64 and cuda_compiler_version in ("10.1", "10.2")]
+    - copy lib\\{{ cuda_compiler_version }}\\cutensor.lib %LIBRARY_LIB%\\  # [win64 and cuda_compiler_version in ("10.1", "10.2")]
+    - copy lib\\11\\cutensor.dll %LIBRARY_BIN%\\                           # [win64 and cuda_compiler_version in ("11.0", )]
+    - copy lib\\11\\cutensor.lib %LIBRARY_LIB%\\                           # [win64 and cuda_compiler_version in ("11.0", )]
   ignore_run_exports_from:
-    - {{ compiler('c') }}    # [linux]
+    - {{ compiler('c') }}     # [linux]
```

Comment:
```suggestion
    - {{ compiler('c') }}     # [linux]
    - {{ compiler('cuda') }}  # [cuda_compiler_version == "11.0"]
```

Reply:
@jakirkham looks like this would make the compilation test fail...

Reply:
...because `cudatoolkit` was not installed at the test stage. But, oddly, the same test passed on Windows!

Reply:
885d4d1 seems to fix it.

Reply:
Weird would have thought `pin_compatible` in `run` would have been sufficient